libnavui-maneuver/src/main/java/com/mapbox/navigation/ui/maneuver/view/MapboxExitText.kt

```diff
@@ -0,0 +1,131 @@
+package com.mapbox.navigation.ui.maneuver.view
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.graphics.Color
+import android.graphics.Typeface
+import android.graphics.drawable.BitmapDrawable
+import android.graphics.drawable.Drawable
+import android.util.AttributeSet
+import androidx.appcompat.widget.AppCompatTextView
+import androidx.core.content.ContextCompat
+import com.mapbox.api.directions.v5.models.BannerComponents
+import com.mapbox.api.directions.v5.models.ManeuverModifier
+import com.mapbox.navigation.ui.base.model.maneuver.ComponentNode
+import com.mapbox.navigation.ui.base.model.maneuver.ExitNumberComponentNode
+import com.mapbox.navigation.ui.maneuver.R
+
+/**
+ * Default Exit View that renders exit number in a specific style.
+ * @property leftDrawable Drawable? denotes the style for exit sign that is on the left.
+ * @property rightDrawable Drawable? denotes the style for exit sign that is on the right.
+ * @property exitBackground Drawable? denotes the exit board style.
+ */
+class MapboxExitText @JvmOverloads constructor(
```

Comment:
How about making all `MapboxManeuverView`'s sub-views `internal` until we understand that there's a use-case for exposing the granular elements?

Reply:
I am not planning to make the sub-views as internal. I want to discuss this pattern with the team and my reasoning behind it, because I feel it is very powerful. However, we have a couple more things to do in maneuvers before it goes GA. I could make them internal if the team doesn't agree with my proposal. I would like to discuss this tomorrow in our discussion for all the other views like TripProgress, SoundButton, RecenterButton etc.

Reply:
Sounds good :+1: